.github/workflows/upgrade-tests.yml

```diff
@@ -85,7 +85,7 @@ jobs:
             env -u RUSTC_WRAPPER \
             env TEST_KINDS="$TEST_KINDS" \
             env FM_DEVIMINT_DISABLE_MODULE_LNV2=1 \
-            runLowPrio ./scripts/tests/upgrade-test.sh "$VERSIONS"
+            ./scripts/tests/upgrade-test.sh "$VERSIONS"
```

Comment:
@bradleystachurski the whole test should not be `runLowPrio`, but we should make sure that all the compilation steps are, so they are graceful for other things like tests that have a deadline.

Reply:
:+1: 

https://github.com/fedimint/fedimint/pull/6720